<?xml version='1.0' encoding='utf-8'?>
<sdf version="1.6">
   <world name="world">
       <plugin name="world_sfm" filename="libWorldSFMPlugin.so">
       </plugin>
       <gui>
           <camera name='gzclient_camera'>
               <pose>-4.70385 10.895 16.2659 -0 0.921795 -1.12701</pose>
           </camera>
       </gui>
       <gravity>0 0 -9.8</gravity>
       <physics default="0" name="default_physics" type="ode">
           <max_step_size>0.001</max_step_size>
           <real_time_factor>1</real_time_factor>
           <real_time_update_rate>1000</real_time_update_rate>
       </physics>
       <scene>
           <ambient>0.6 0.6 0.6 1</ambient>
           <background>0.4 0.4 0.4 1</background>
           <shadows>false</shadows>
       </scene>
       <include>
           <uri>model://sun</uri>
       </include>
       <include>
           <uri>model://ground_plane</uri>
           <pose>0 0 -0.1 0 -0 0</pose>
       </include>
       <model name="circular_workspace">
           <allow_auto_disable>false</allow_auto_disable>
           <static>true</static>
           <pose>0 0 -0.1 0 0 0</pose>
           <static>true</static>
           <link name="CW_link">
               <collision name="CW_collision">
                   <pose>0 0 0 0 0 0</pose>
                   <geometry>
                       <mesh>
                       <uri>model://circular_workspace_7m.dae</uri>
                       <scale>1 1 1</scale>
                       </mesh>
                   </geometry>
               </collision>
               <visual name="CW_visual">
                   <pose>0 0 0 0 0 0</pose>
                   <geometry>
                       <mesh>
                           <uri>model://circular_workspace_7m.dae</uri>
                           <scale>1 1 1</scale>
                       </mesh>
                   </geometry>
               </visual>
           </link>
       </model>
       <actor name="agent1">
          <pose>0 0 1.01 0 0 0</pose>
          <skin>
              <filename>stand.dae</filename>
              <scale>1.0</scale>
          </skin>
          <animation name="walking">
              <filename>walk.dae</filename>
              <scale>1.000000</scale>
              <interpolate_x>true</interpolate_x>
          </animation>
       </actor>
       <actor name="agent2">
          <pose>0 1 1.01 0 0 0</pose>
          <skin>
              <filename>stand.dae</filename>
              <scale>1.0</scale>
          </skin>
          <animation name="walking">
              <filename>walk.dae</filename>
              <scale>1.000000</scale>
              <interpolate_x>true</interpolate_x>
          </animation>
       </actor>
       <actor name="agent3">
          <pose>0 2 1.01 0 0 0</pose>
          <skin>
              <filename>stand.dae</filename>
              <scale>1.0</scale>
          </skin>
          <animation name="walking">
              <filename>walk.dae</filename>
              <scale>1.000000</scale>
              <interpolate_x>true</interpolate_x>
          </animation>
       </actor>
       <actor name="agent4">
          <pose>0 3 1.01 0 0 0</pose>
          <skin>
              <filename>stand.dae</filename>
              <scale>1.0</scale>
          </skin>
          <animation name="walking">
              <filename>walk.dae</filename>
              <scale>1.000000</scale>
              <interpolate_x>true</interpolate_x>
          </animation>
       </actor>
       <actor name="agent5">
          <pose>0 4 1.01 0 0 0</pose>
          <skin>
              <filename>stand.dae</filename>
              <scale>1.0</scale>
          </skin>
          <animation name="walking">
              <filename>walk.dae</filename>
              <scale>1.000000</scale>
              <interpolate_x>true</interpolate_x>
          </animation>
       </actor>
       <actor name="agent6">
          <pose>0 5 1.01 0 0 0</pose>
          <skin>
              <filename>stand.dae</filename>
              <scale>1.0</scale>
          </skin>
          <animation name="walking">
              <filename>walk.dae</filename>
              <scale>1.000000</scale>
              <interpolate_x>true</interpolate_x>
          </animation>
       </actor>
       <actor name="agent7">
          <pose>0 6 1.01 0 0 0</pose>
          <skin>
              <filename>stand.dae</filename>
              <scale>1.0</scale>
          </skin>
          <animation name="walking">
              <filename>walk.dae</filename>
              <scale>1.000000</scale>
              <interpolate_x>true</interpolate_x>
          </animation>
       </actor>
       <actor name="agent8">
          <pose>0 7 1.01 0 0 0</pose>
          <skin>
              <filename>stand.dae</filename>
              <scale>1.0</scale>
          </skin>
          <animation name="walking">
              <filename>walk.dae</filename>
              <scale>1.000000</scale>
              <interpolate_x>true</interpolate_x>
          </animation>
       </actor>
       <actor name="agent9">
          <pose>0 8 1.01 0 0 0</pose>
          <skin>
              <filename>stand.dae</filename>
              <scale>1.0</scale>
          </skin>
          <animation name="walking">
              <filename>walk.dae</filename>
              <scale>1.000000</scale>
              <interpolate_x>true</interpolate_x>
          </animation>
       </actor>
       <actor name="agent10">
          <pose>0 9 1.01 0 0 0</pose>
          <skin>
              <filename>stand.dae</filename>
              <scale>1.0</scale>
          </skin>
          <animation name="walking">
              <filename>walk.dae</filename>
              <scale>1.000000</scale>
              <interpolate_x>true</interpolate_x>
          </animation>
       </actor>
       <actor name="agent11">
          <pose>0 10 1.01 0 0 0</pose>
          <skin>
              <filename>stand.dae</filename>
              <scale>1.0</scale>
          </skin>
          <animation name="walking">
              <filename>walk.dae</filename>
              <scale>1.000000</scale>
              <interpolate_x>true</interpolate_x>
          </animation>
       </actor>
       <actor name="agent12">
          <pose>0 11 1.01 0 0 0</pose>
          <skin>
              <filename>stand.dae</filename>
              <scale>1.0</scale>
          </skin>
          <animation name="walking">
              <filename>walk.dae</filename>
              <scale>1.000000</scale>
              <interpolate_x>true</interpolate_x>
          </animation>
       </actor>
       <actor name="agent13">
          <pose>0 12 1.01 0 0 0</pose>
          <skin>
              <filename>stand.dae</filename>
              <scale>1.0</scale>
          </skin>
          <animation name="walking">
              <filename>walk.dae</filename>
              <scale>1.000000</scale>
              <interpolate_x>true</interpolate_x>
          </animation>
       </actor>
       <actor name="agent14">
          <pose>0 13 1.01 0 0 0</pose>
          <skin>
              <filename>stand.dae</filename>
              <scale>1.0</scale>
          </skin>
          <animation name="walking">
              <filename>walk.dae</filename>
              <scale>1.000000</scale>
              <interpolate_x>true</interpolate_x>
          </animation>
       </actor>
       <actor name="agent15">
          <pose>0 14 1.01 0 0 0</pose>
          <skin>
              <filename>stand.dae</filename>
              <scale>1.0</scale>
          </skin>
          <animation name="walking">
              <filename>walk.dae</filename>
              <scale>1.000000</scale>
              <interpolate_x>true</interpolate_x>
          </animation>
       </actor>
       <actor name="agent16">
          <pose>0 15 1.01 0 0 0</pose>
          <skin>
              <filename>stand.dae</filename>
              <scale>1.0</scale>
          </skin>
          <animation name="walking">
              <filename>walk.dae</filename>
              <scale>1.000000</scale>
              <interpolate_x>true</interpolate_x>
          </animation>
       </actor>
       <actor name="agent17">
          <pose>0 16 1.01 0 0 0</pose>
          <skin>
              <filename>stand.dae</filename>
              <scale>1.0</scale>
          </skin>
          <animation name="walking">
              <filename>walk.dae</filename>
              <scale>1.000000</scale>
              <interpolate_x>true</interpolate_x>
          </animation>
       </actor>
       <actor name="agent18">
          <pose>0 17 1.01 0 0 0</pose>
          <skin>
              <filename>stand.dae</filename>
              <scale>1.0</scale>
          </skin>
          <animation name="walking">
              <filename>walk.dae</filename>
              <scale>1.000000</scale>
              <interpolate_x>true</interpolate_x>
          </animation>
       </actor>
       <actor name="agent19">
          <pose>0 18 1.01 0 0 0</pose>
          <skin>
              <filename>stand.dae</filename>
              <scale>1.0</scale>
          </skin>
          <animation name="walking">
              <filename>walk.dae</filename>
              <scale>1.000000</scale>
              <interpolate_x>true</interpolate_x>
          </animation>
       </actor>
       <actor name="agent20">
          <pose>0 19 1.01 0 0 0</pose>
          <skin>
              <filename>stand.dae</filename>
              <scale>1.0</scale>
          </skin>
          <animation name="walking">
              <filename>walk.dae</filename>
              <scale>1.000000</scale>
              <interpolate_x>true</interpolate_x>
          </animation>
       </actor>
       <actor name="agent21">
          <pose>0 20 1.01 0 0 0</pose>
          <skin>
              <filename>stand.dae</filename>
              <scale>1.0</scale>
          </skin>
          <animation name="walking">
              <filename>walk.dae</filename>
              <scale>1.000000</scale>
              <interpolate_x>true</interpolate_x>
          </animation>
       </actor>
       <actor name="agent22">
          <pose>0 21 1.01 0 0 0</pose>
          <skin>
              <filename>stand.dae</filename>
              <scale>1.0</scale>
          </skin>
          <animation name="walking">
              <filename>walk.dae</filename>
              <scale>1.000000</scale>
              <interpolate_x>true</interpolate_x>
          </animation>
       </actor>
       <actor name="agent23">
          <pose>0 22 1.01 0 0 0</pose>
          <skin>
              <filename>stand.dae</filename>
              <scale>1.0</scale>
          </skin>
          <animation name="walking">
              <filename>walk.dae</filename>
              <scale>1.000000</scale>
              <interpolate_x>true</interpolate_x>
          </animation>
       </actor>
       <actor name="agent24">
          <pose>0 23 1.01 0 0 0</pose>
          <skin>
              <filename>stand.dae</filename>
              <scale>1.0</scale>
          </skin>
          <animation name="walking">
              <filename>walk.dae</filename>
              <scale>1.000000</scale>
              <interpolate_x>true</interpolate_x>
          </animation>
       </actor>
       <actor name="agent25">
          <pose>0 24 1.01 0 0 0</pose>
          <skin>
              <filename>stand.dae</filename>
              <scale>1.0</scale>
          </skin>
          <animation name="walking">
              <filename>walk.dae</filename>
              <scale>1.000000</scale>
              <interpolate_x>true</interpolate_x>
          </animation>
       </actor>
       <actor name="agent26">
          <pose>0 25 1.01 0 0 0</pose>
          <skin>
              <filename>stand.dae</filename>
              <scale>1.0</scale>
          </skin>
          <animation name="walking">
              <filename>walk.dae</filename>
              <scale>1.000000</scale>
              <interpolate_x>true</interpolate_x>
          </animation>
       </actor>
       <actor name="agent27">
          <pose>0 26 1.01 0 0 0</pose>
          <skin>
              <filename>stand.dae</filename>
              <scale>1.0</scale>
          </skin>
          <animation name="walking">
              <filename>walk.dae</filename>
              <scale>1.000000</scale>
              <interpolate_x>true</interpolate_x>
          </animation>
       </actor>
       <actor name="agent28">
          <pose>0 27 1.01 0 0 0</pose>
          <skin>
              <filename>stand.dae</filename>
              <scale>1.0</scale>
          </skin>
          <animation name="walking">
              <filename>walk.dae</filename>
              <scale>1.000000</scale>
              <interpolate_x>true</interpolate_x>
          </animation>
       </actor>
       <actor name="agent29">
          <pose>0 28 1.01 0 0 0</pose>
          <skin>
              <filename>stand.dae</filename>
              <scale>1.0</scale>
          </skin>
          <animation name="walking">
              <filename>walk.dae</filename>
              <scale>1.000000</scale>
              <interpolate_x>true</interpolate_x>
          </animation>
       </actor>
       <actor name="agent30">
          <pose>0 29 1.01 0 0 0</pose>
          <skin>
              <filename>stand.dae</filename>
              <scale>1.0</scale>
          </skin>
          <animation name="walking">
              <filename>walk.dae</filename>
              <scale>1.000000</scale>
              <interpolate_x>true</interpolate_x>
          </animation>
       </actor>
       <actor name="agent31">
          <pose>0 30 1.01 0 0 0</pose>
          <skin>
              <filename>stand.dae</filename>
              <scale>1.0</scale>
          </skin>
          <animation name="walking">
              <filename>walk.dae</filename>
              <scale>1.000000</scale>
              <interpolate_x>true</interpolate_x>
          </animation>
       </actor>
       <actor name="agent32">
          <pose>0 31 1.01 0 0 0</pose>
          <skin>
              <filename>stand.dae</filename>
              <scale>1.0</scale>
          </skin>
          <animation name="walking">
              <filename>walk.dae</filename>
              <scale>1.000000</scale>
              <interpolate_x>true</interpolate_x>
          </animation>
       </actor>
       <actor name="agent33">
          <pose>0 32 1.01 0 0 0</pose>
          <skin>
              <filename>stand.dae</filename>
              <scale>1.0</scale>
          </skin>
          <animation name="walking">
              <filename>walk.dae</filename>
              <scale>1.000000</scale>
              <interpolate_x>true</interpolate_x>
          </animation>
       </actor>
       <actor name="agent34">
          <pose>0 33 1.01 0 0 0</pose>
          <skin>
              <filename>stand.dae</filename>
              <scale>1.0</scale>
          </skin>
          <animation name="walking">
              <filename>walk.dae</filename>
              <scale>1.000000</scale>
              <interpolate_x>true</interpolate_x>
          </animation>
       </actor>
       <actor name="agent35">
          <pose>0 34 1.01 0 0 0</pose>
          <skin>
              <filename>stand.dae</filename>
              <scale>1.0</scale>
          </skin>
          <animation name="walking">
              <filename>walk.dae</filename>
              <scale>1.000000</scale>
              <interpolate_x>true</interpolate_x>
          </animation>
       </actor>
   </world>
</sdf>